DEVELOPMENT.md

```diff
@@ -425,29 +425,41 @@ build times. If you need to update this image, proceed as follows:
    create a new branch and build the updated image.
 
 ## Releases
-
 ### Prerequisites
+#### Google Cloud CLI
+
+For uploading artifacts to releases.radicle.xyz, you'll need a working `gcloud`
+environment. To do so, follow points 1 and 2 from the
+[Docker image updates][do] section.
+
 
 #### GitHub `hub` CLI tool
 
-Please install the [`hub`][hb] CLI tool, we use it in our release automation
-script to:
+Please install the [`hub`][hb] CLI tool (version >= **2.14**), we use it in our
```

Comment:
Can we check this in the release script?

Reply:
283ec55da6569292417da9b4cbaea93083b14029

I removed the check for `git` because it's a pre-requisite of `hub`.

Error when `hub` missing:
```
$ scripts/release.ts

Could not find the GitHub CLI tool >= 2.14.1.
You can get it here:

  https://github.com/github/hub

error Command failed with exit code 1.
```

Error when it's the wrong version:
```
$ scripts/release.ts

Please upgrade your hub CLI tool to the minimum required version of 2.14.1.

error Command failed with exit code 1.
```

Reply:
> Could not find the GitHub CLI

This bit is confusing since there is a Github cli tool, the official alternative to `hub`, right?

Reply:
> This bit is confusing since there is a Github cli tool, the official alternative to hub, right?

Whoops, you're right!

dbd97591e885e4de733eecc001720194ff77b701

Reply:
I wonder why we went with the unofficial `hub` CLI to begin with?